assignments/array-methods.js

```diff
@@ -54,27 +54,58 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+// The event director needs both the first and last names of each runner for their running bibs.  
+//Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach(x => { 
+  let name = (x.first_name);
+  let lastname= (x.last_name);
+  fullName.push(name.concat(" " + lastname));
```

Comment:
I like this use of concat. Clever. Most people are just gonna add them together with a space in between. Nice job.